app/contributors/[slug]/page.tsx

```diff
@@ -50,7 +49,7 @@ export default async function Contributor({ params }: Params) {
       <section className="bg-gray-200 px-4 py-8 dark:bg-gray-800">
         <div className="mx-auto flex max-w-6xl flex-col md:flex-row ">
           <div className="md:w-2/3">
-            <InfoCard contributor={contributor} />
+            <InfoCard contributor={contributor} isRank={true} />
```

Comment:
Rely on the `minimal` prop instead

```suggestion
            <InfoCard contributor={contributor} />
```

Reply:
we are using githubevent component in home page also so I use props to check rank is shown or not

Reply:
GitHub event has nothing to do with InfoCard IIRC

Reply:
Sorry, I mean InfoCard we are showing in the home page also, 
![image](https://github.com/coronasafe/leaderboard/assets/99136186/25c3d6e1-b6c2-4227-83c2-74a5bc7d101d)

Reply:
Yea, `minimal` is the prop that differentiates from whats shown in home page and whats shown in contributor profile.

If minimal is true, skip showing rank